graylog2-server/src/main/java/org/graylog2/dashboards/widgets/SearchResultChartWidget.java

```diff
@@ -72,12 +61,8 @@ public TimeRange getTimeRange() {
     public Map<String, Object> getPersistedConfig() {
         final ImmutableMap.Builder<String, Object> persistedConfig = ImmutableMap.<String, Object>builder()
                 .put("query", query)
-                .put("interval", interval.toString().toLowerCase())
-                .put("timerange", timeRange.getPersistedConfig());
-
-        if (!isNullOrEmpty(streamId)) {
-            persistedConfig.put("stream_id", streamId);
-        }
+                .put("timerange", timeRange.getPersistedConfig())
+                .putAll(super.getPersistedConfig());
```

Comment:
Shouldn't this be the first addition to the map so that the query parameters and the local config can overwrite settings?